src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -45,5 +54,6 @@ public Command setCenterRollerSpeed(MutableMeasure<Velocity<Angle>> speed) {
     public void periodic() {
         io.updateInputs();
         Logger.processInputs(this.getClass().getName(), inputs);
+        intakeLigament.setAngle(inputs.currentAngle.in(Units.Radians));
```

Comment:
The `setAngle` function for the ligament takes degrees.

Reply:
OK

Reply:
423a178e32ada435b677b785a6c731fd229beeee